data/json/items/containers.json

```diff
@@ -2696,12 +2696,13 @@
     "id": "aquarium_medium",
     "type": "GENERIC",
     "category": "container",
-    "name": { "str": "glass aquarium" },
-    "description": "A 50 gallon glass aquarium, used for safely storing small animals for use as pets and food.",
-    "weight": "83 kg",
+    "name": { "str": "50 gallon aquarium" },
```

Comment:
Is this consistent with container naming? AFAIK other containers list their volume in liters (3L jar, steel drum (100L), vehicle tank (60L))

Reply:
IIRC at least one drum is listed as <amount> gallon drum

Reply:
No, no, no. The drums reference gallons in the IDs whereas in their names they are litres. I went with American sizes for Aquariums as the game is set in New England and, sure, I could instead call it a 189-litre aquarium but it doesn't roll off the tongue the same way.

Reply:
Okay after asking around in the discord it appears that this shouldn't be an issue as that is exactly what the aquariums are named in New England. American aquariums reference gallons in their names so these, which are also American, referencing gallons should not be a problem.